data/human/boarding missions.txt

```diff
@@ -600,3 +600,238 @@ mission "Turn in Tarquin"
+			`	However, you soon notice a blinking light on the controls of the ship, it appears to be some kind of beacon. But what could it be signaling? The arrival of pirate ships entering the system gives you a pretty good guess as you race back to your ship.`
+				launch
+			label virus
+			action
+				payment -200000
+			`	However, moments later a screen lights up in the cockpit displaying something strange. Upon closer inspection you realize it's some form of malicious software, and it's attempting to hack into your ship!`
+			`	You race back to your ship, quickly launching and destroying the <origin>, but by the time you manage to destroy the virus it has already hacked into your bank account and stolen <payment> credits from you.`
+				depart
+			label salvage
+			`	Deciding not to risk it you quickly scour the ship for valuable scrap and outfits.`
+			branch valuable
+				random < 40
+			action
+				payment 22500
+			`	Upon a brief inspection of the ship you find little valuable, only managing to salvage a meager <payment> credits from the <origin>. Before you depart you detonate the ship to prevent any other derelict hunters from scouring an empty ship.`
+				depart
+			label valuable
+			branch rare
+				or
+					and
+						random < 25
+						or
+							not "derelict treasure map"
+							not "derelict family investment"
+					random < 5
+			action
+				payment 63000
+			`	Searching through the ship you find that it is quite valuable and you manage to salvage a substantital amount from the <origin>, worth <payment> credits. Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
+				depart
+			label rare
+			branch quest
+				or
+					not "derelict treasure map"
+					not "derelict family investment"
+			action
+				payment 475000
+			`	Hidden away under a panel you find a small box. Opening it reveals a huge amount of credits worth <payment> credits! Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
+				depart
+			label quest
+			branch treasure
+				not "derelict treasure map"
+				or
+					random < 50
+					has "derelict family investment"
+			action
+				set "derelict family investment"
+			`	As you search the ship for anything valuable you find a large black case filled with credits, investigating further you find a text log next to the case, you decide to take both of them with you. Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
+				depart
+			label treasure
+			action
+				set "derelict treasure map"
+				payment 26500
+			`	As you search the ship for anything valuable you find some kind of map tucked into a panel, it's easy to carry, allowing you time to further salvage a meager <payment> credits from the <origin>. Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
```

Comment:
Same here.
```suggestion
			`	As you search the ship for anything valuable you find some kind of map tucked into a panel. It's easy to carry, allowing you time to further salvage a meager <payment> credits from the <origin>. Before you depart you detonate the ship to prevent any other derelict hunters from scouring the now empty ship.`
```